src/client/src/styleguide/components/Input.tsx

```diff
@@ -111,9 +111,11 @@ const Label = styled.label<LabelProps>`
         ? theme.secondary[2]
         : theme.secondary[2]
       : disabled
-      ? theme.colors.light.secondary[1]
-      : theme.colors.light.secondary[2]};
+      ? theme.colors.dark.secondary[1]
```

Comment:
weird .. good catch .. any idea why above  is theme.secondary and not theme.light.secondary? and why are both parts of ternary the same??

Reply:
and what's the point in setting a theme if you need to apply these nested ternaries ... some further discussion for the UI Kit integration debate .. will add stuff to that ticket